src/pages/ga4/enhanced-ecommerce/index.tsx

```diff
@@ -0,0 +1,57 @@
+// Copyright 2022 Google Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import * as React from "react"
+
+import Layout from "@/components/Layout"
+import {graphql} from "gatsby";
+import {Header} from "@/components/ga4/EnhancedEcommerce/header";
+import {ProductListing} from "@/components/ga4/EnhancedEcommerce/product-listing";
+import {Footer} from "@/components/ga4/EnhancedEcommerce/footer";
+
+export default ({ location: { pathname }, data  }) => {
+    return (
+        <Layout
```

Comment:
In order to keep the file layout consistent, I recommend following the pattern in the other pages where `Layout` has one direct child that's the entire component tree. See [hit builder](https://github.com/googleanalytics/ga-dev-tools/blob/main/src/pages/hit-builder.tsx#L28) as an example.